examples/website/texture-tester/app.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, {PureComponent} from 'react';
+import {render} from 'react-dom';
+import {instrumentGLContext, Program} from '@luma.gl/core';
+import TexturesBlock from './components/textures-block';
+import {IMAGES_DATA} from './textures-data';
+
+// TEXTURE SHADERS
+
+const vs = `
+precision highp float;
+
+attribute vec2 position;
+varying vec2 uv;
+
+void main() {
+  gl_Position = vec4(position, 0.0, 1.0);
+  uv = vec2(position.x * .5, -position.y * .5) + vec2(.5, .5);
+}
+`;
+
+const fs = `
+precision highp float;
+
+uniform sampler2D tex;
+varying vec2 uv;
+
+void main() {
+  gl_FragColor = vec4(texture2D(tex, uv).rgb, 1.);
+}
+`;
+
+export default class App extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      canvas: null,
+      gl: null,
+      program: null
+    };
+
+    this.setupCanvas = this.setupCanvas.bind(this);
```

Comment:
Why are you binding these? Are you using them as callbacks? If not you don't need to bind them.